Source/FluidOscServer.cpp

```diff
@@ -28,6 +28,7 @@ void FluidOscServer::oscBundleReceived(const juce::OSCBundle &bundle) {
     selectedMidiClip = nullptr;
     selectedAudioTrack = nullptr;
     selectedPlugin = nullptr;
+    selectedWaveClip = nullptr;
```

Comment:
This was deleted from the header, so its presence breaks complication. It's best practice to always ensure that your code compiles before it ends up in the repository.